v2/internal/attacktechniques/entra-id/persistence/guest-user/main.go

```diff
@@ -0,0 +1,179 @@
+package azure
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+	graphmodels "github.com/microsoftgraph/msgraph-sdk-go/models"
+	"log"
+	"strings"
+	entra_id_utils "github.com/datadog/stratus-red-team/v2/internal/utils/entra_id"
+)
+
+const codeBlock = "```"
+const AttackTechniqueId = "entra-id.persistence.guest-user"
+
+func init() {
+	stratus.GetRegistry().RegisterAttackTechnique(&stratus.AttackTechnique{
+		ID:           "entra-id.persistence.guest-user",
+		FriendlyName: "Create Guest User for Persistent Access",
+		Description: `
+An attacker can abuse the Guest User invite process to gain persistent access to an environment, as they can invite themselves as a guest.
+
+Warm-up:
+
+- N/A, this technique does not have a warm-up stage
+
+Detonation:
+
+- Invite Guest User
+
+References:
+
+- https://securitylabs.datadoghq.com/cloud-security-atlas/attacks/inviting-external-users/
+
+!!! note
+
+	Since the target e-mail must exist for this attack simulation to work, Stratus Red Team grants the role to ` + entra_id_utils.DefaultFictitiousAttackerEmail + ` by default.
```

Comment:
```suggestion
	Since the target e-mail must exist for this attack simulation to work, Stratus Red Team creates a guest user with the e-mail ` + entra_id_utils.DefaultFictitiousAttackerEmail + ` by default.
```